pkg/graph/schema.graphql

```diff
@@ -1416,6 +1416,11 @@ type DateHistogramAggregationBucket {
   minModifiedDts: Time!
 }
 
+input SearchFilter {
+  type: SearchFilterType!
+  value: Any!
+}

```

Comment:
Per our convo yesterday, I think it would greatly help the clarity of this schema to type the values for the different kinds of filters you can apply (or, at least document it in a comment on this input type)

For example, constructing a `CHANGED_AFTER` search - What format should I use for the date? Same as `scalar Time`? RFC3339? For a `MODEL_PLAN_SECTION`, should I pass an array of sections? Should they be strings with the same value as this enum? https://github.com/CMSgov/mint-app/blob/e4a024fc0a4af0e65bae9e51bd9b6b76759845ff/pkg/graph/schema.graphql#L1644-L1652

These are the kinds of questions that are helped by more strict typing, but in the short term if we can just document the expected input types and formats, that can be sufficient to push this through for the FE in my opinion

Reply:
As discussed, further work on this is ticketed in EASI-2999.